DataFormats/CaloTowers/interface/CaloTower.h

```diff
@@ -62,7 +62,7 @@ class CaloTower : public reco::LeafCandidate {
    // setters
   void addConstituent( DetId id ) { constituents_.push_back( id ); }
   void addConstituents( const std::vector<DetId>& ids );
-  void setConstituents( std::vector<DetId>&& ids ) { constituents_=std::move(ids);}
+  void setConstituents( std::vector<DetId>& ids ) { constituents_=ids;}
```

Comment:
If I understand correctly, there is extra cost to this,
to be evaluated.
If needed in cling, appropriate ifdefs can deliver needed content.
Also, there should be a const qualifier here